src/handler.ts

```diff
@@ -1,8 +1,8 @@
 const redirectHttpCode = 301
 const redirectMap = new Map([
-  ['/discord', 'https://discord.com/invite/hqBNg4u'],
-  ['/reddit', 'https://reddit.com/r/IndiaInvestments'],
-  ['/youtube', 'https://www.youtube.com/channel/UC324pobafJz5J_KdABVh4Tw'],
+  ['/discord', '${{ secrets.BOT_GITHUB_TOKEN }}'],
```

Comment:
No, cannot be accessed like this. Also, this code is not valid TypeScript. Because `${{ }}` has no meaning in TS.

It should be accessed in code as `process.env.BOT_GITHUB_TOKEN`.

In Webpack config file, one has to map the secret to this environment variable.

Reply:
Updated the config, using `${process.env}` type variables instead to follow proper TS conventions.